HarpGuru/Controls/CovarianceSeries/getHarpKeyControllers/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { HarpKeyControllers, PozitionIds, PitchIds } from 'harpstrata'
+
+import type { HarpKeyCovariancePrimer } from '../types'
+import { CovariantMembers } from '../types'
+
+export const getHarpKeyControllers = (props: HarpKeyCovariancePrimer): HarpKeyControllers => {
+  const { lockedType, lockedValue, variedValue } = props
+
+  if ( lockedType === CovariantMembers.RootPitch ) {
+    const pozitionId = variedValue as PozitionIds
```

Comment:
More type assertions here. I would like to explore a way to make them unnecessary.

Reply:
https://github.com/js-jslog/harpnative/issues/11 raised